internal/svcctl/svcctl.go

```diff
@@ -160,34 +165,43 @@ func stopAndWait(ctx context.Context, ipComm IPCommunicator) error {
 }
 
 func waitDown(ctx context.Context, ipComm IPCommunicator) error {
+	start := time.Now()
 	for try := 1; try <= 32; try++ {
 		select {
 		case <-ctx.Done():
 			return ctx.Err()
 		default:
 		}
 
-		start := time.Now()
+		tryStart := time.Now()
 		timeout := time.Millisecond * time.Duration(try*try)
 
-		logging.Debug("Attempt: %d, timeout: %v", try, timeout)
+		logging.Debug("Attempt: %d, timeout: %v, total: %v", try, timeout, time.Since(start))
 		if err := ping(ctx, ipComm, timeout); err != nil {
-			if errors.Is(err, errNotUp) {
+			// The current error (e.g. timeout) does not reveal enough info, try again.
```

Comment:
```suggestion
			// The current error (e.g. timeout) does not reveal enough info, try again.
			// We don't need to sleep for this type of error because by definition this is a timeout, and time has already elapsed.
```